create_patched_sdk.sh

```diff
@@ -1,182 +1,236 @@
 #!/usr/bin/env bash
 
 print_usage() {
-    printf "Usage: $(basename $0) {use_simulator} {sdks_output_path} {overwrite_existing} {no_warnings} {tbd_tool_path} {xcode_installation_path}\n"
-    printf "Note: {} options are optional to provide, and can be ignored with a '-'\n"
+cat << EOF
+Usage: $(basename $0) {use_simulator} {sdks_output_path} {no_overwrite} {no_warnings} {tbd_tool} {xcode_installation_path} {sdk_platform}
+Options:
+  - use_simulator (bool)
+	Dump binaries from simulator runtime as opposed to DeviceSupport files
+  - sdks_output_path (str)
+ 	Directory to install SDK to (default: '\$THEOS/sdks')
+  - no_overwrite (bool)
+	Disable overwrite of existing dumped binaries (default: 0)
+  - no_warnings (bool)
+	Disable 'tbd' warnings (default: 0)
+  - tbd_tool (str)
+	Path to 'tbd' tool from inoahdev (default: 'tbd')
+  - xcode_installation_path (str)
+	Path to target Xcode version (default: currently selected Xcode.app)
+  - sdk_platform (str)
+	Target platform for patched SDK (default: 'iOS'; alternatively 'tvOS')
+Note: {} options are optional to provide, and can be ignored with a '-'
+EOF
 }
 
 ignored() {
-    if [[ $1 == "-" ]]; then
-        return 0
-    else
-        return 1
-    fi
+	if [[ $1 == "-" ]]; then
+		return 0
+	else
+		return 1
+	fi
 }
 
-if [[ $1 == "-h" ]]; then
-    print_usage
-    exit 0
+if [[ $# -lt 1 ]] || [[ $1 == "-h" ]] || [[ $1 == "--help" ]]; then
+	print_usage
+	exit 0
+fi
+
+sdk_platform="$7"
```

Comment:
nit: we should validate the `sdk_platform` is one we expect (currently "iOS", "tvOS") to avoid confusion

Reply:
Good call. Unsure if the `tr` method works on OSX, but I believe we use it in theos/theos (?), so it should be fine?

Reply:
Just checked - works well